builds/aks-cleanup/preview-scheduled-cleanup.yml

```diff
@@ -4,8 +4,8 @@ name: AKS Cleanup Preview
 trigger: none
 pr: none
 schedules:
-  - cron: '0 8-18 * * 1-5'
-    displayName: Every working hour
+  - cron: '0 */12 * * *'
+    displayName: Clean up every 12 hours
```

Comment:
I'm assuming this has been agreed with Praveen, so my question is more for him than anyone else.

The previous implementation could have caused deletion of pods happening at any hour of the day, which could force people to relaunch a build in the middle of the day (or at any hour really).

This change will see the cleanup run at 12:00 and at 00:00. So even after this change people may still be forced to re-run their builds in the middle of the day, adding to the strain on Jenkins.

Can we do `0 19-23 * * *` instead? This way people would be able to work without disruption during the day and the cleanup would happen after hours.

Reply:
`0 19-23,0-7 * * 1-5 ` For every one hour between 7pm - 7am between Monday to Friday

Reply:
I would leave it to run every 1 hour. If a PR is not updated for 3 working days and is being used for testing, its probably a ways of working improvement that's needed.

Reply:
After a talk with Praveen, here's what we do:
- we restore the original cron schedule
- if for some reason it turns out we need to go back to the 1 day exception for civil, we also revise the cron schedule to run at the end of the day. At that stage however we'll have to do a whole bunch of other changes as this script is shared across the whole HMCTS.